docs/integrations/aws.md

```diff
@@ -433,6 +446,25 @@ spark-sql --conf spark.sql.catalog.my_catalog=org.apache.iceberg.spark.SparkCata
 ```
 For the above example, the objects in S3 will be saved with tags: `my_key1=my_val1` and `my_key2=my_val2`.
```

Comment:
I think we might need to call out here that these tags specified will be used only in object creation like we called out that delete tags will be used in tagging in delete operation

Reply:
I have added the details.